django_prices_vatlayer/utils.py

```diff
@@ -0,0 +1,73 @@
+from prices import LinearTax
+import requests
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured, ObjectDoesNotExist
+from django.utils import six
+
+from .models import VAT, RateTypes, DEFAULT_TYPES_INSTANCE_ID
+
+try:
+    ACCESS_KEY = settings.VATLAYER_ACCESS_KEY
+except AttributeError:
+    raise ImproperlyConfigured('VATLAYER_ACCESS_KEY is required')
+
+USE_HTTPS = getattr(settings, 'USE_HTTPS', False)
```

Comment:
How about we allow you to override the API endpoint instead? I'm thinking something like `VATLAYER_API = getattr(settings, 'VATLAYER_API', 'http://apilayer.net/api/')`